lib/SPIRV/libSPIRV/spirv_internal.hpp

```diff
@@ -91,10 +84,7 @@ enum InternalExecutionMode {
   IExecModeStreamingInterfaceINTEL = 6154
 };
 
-enum InternalLoopControlMask {
-  ILoopControlLoopCountINTELMask = 0x1000000,
-  ILoopControlMaxReinvocationDelayINTELMask = 0x2000000
-};
+enum InternalLoopControlMask {};
```

Comment:
Could you please clarify why we still need this trick?

Reply:
We don't need it, but we might add new Masks here in the future, so I don't want to remove enum

Reply:
I see your point, but it still looks strange to me. There is a chance that this line would not be found by a developer and a new enum will be created.

Reply:
I see your point, but it still looks strange to me. There is a chance that this line would not be found by a developer and a new enum will be created.

Reply:
Agree, removed the enum